core/impl/src/main/java/com/blazebit/persistence/impl/EntityMetamodelImpl.java

```diff
@@ -595,7 +608,12 @@ public AttributeEntry(JpaProvider jpaProvider, ManagedType<X> ownerType, Attribu
             JpaProvider.ConstraintType[] requiresTreatFilter = new JpaProvider.ConstraintType[joinTypes.length];
             if (ownerType instanceof EntityType<?>) {
                 EntityType<?> entityType = (EntityType<?>) ownerType;
-                this.isForeignJoinColumn = jpaProvider.isForeignJoinColumn(entityType, attributeName);
+                try {
+                    this.isForeignJoinColumn = jpaProvider.isForeignJoinColumn(entityType, attributeName);
+                }
+                catch (Exception e) {
```

Comment:
I suppose you will remove this later and this is just for testing?

Reply:
Yes, should not have been comitted.